server/src/routers/users.ts

```diff
@@ -12,7 +12,7 @@ import {
 export default function usersRouter(router: express.Router) {
   router.get("/users", getUsers);
   router.get("/users/:id", getUser);
-  router.post("/user", postUser);
```

Comment:
It should be corrected back to user (single user) without the s

Reply:
it's corrected now. Thanks for your collaboration